scripts/us_census/acs5yr/subject_tables/common/base.py

```diff
@@ -0,0 +1,87 @@
+"""Base data loader for ACS subject tables"""
+
+import os
+import csv
+import json
+import pandas as pd
+
+from io import BytesIO
+from requests import get
+from zipfile import ZipFile
+
+import generateColMap
+
+
+_TEMPLATE_TMCF = """Node: E:SubjectTable->E{index}
+typeOf: dcs:StatVarObservation
+variableMeasured: C:SubjectTable->dcs:{statVar}
+measurementMethod: dcs:CesnsusACS{estimatePeriod}yrSurvey
```

Comment:
1. typo
2. this should be something different than CensusACS5yrSurvey... I thought we said CensusACS5yrSurvey_SubjectTables or something

Reply:
1. Fixed
2. Sure, I can make the change. But this change will affect the current [S2201 import](https://autopush.datacommons.org/browser/geoId/06?statVar=Count_Household_WithFoodStampsInThePast12Months) which uses [`CensusACS5yrSurvey`](https://github.com/datacommonsorg/data/blob/master/scripts/us_census/acs5yr/subject_tables/common/process.py#L23) as the `measurementMethod`. I assume, this was done because `CensusACS5yrSurvey` was an existing  `measurementMethod` in DCS.

Reply:
If we use a separate TMCF file, not generated by code, then S2201 can use a separate one from the rest?

Reply:
Yes! I think that should work.